pkg/jx/cmd/create_git_api_token.go

```diff
@@ -112,12 +102,6 @@ func (o *CreateGitTokenOptions) Run() error {
 		userAuth.ApiToken = o.ApiToken
 	}
 
-	tokenUrl := gits.ProviderAccessTokenURL(server.Kind, server.URL, userAuth.Username)
-
-	if userAuth.IsInvalid() && o.Password != "" {
-		err = o.tryFindAPITokenFromBrowser(tokenUrl, userAuth)
```

Comment:
If I understand correctly this is not “dead code” you are removing.

I know nothing about this usage of Chrome. I did remove the _other_ usage, for Jenkins, in #2651.

Reply:
Thank you @jglick for the feedback. I had to handle some internal issues.

Instead of referring to the whole change as dead code removal (in this case: code that should not or cannot work on anyone's machine and refers to a set of dependencies that do not exist generally), I may say this is in the same line of thought as #2651.

This creates a Chrome instance using a no-longer-existing-or-supported API set from ChromeDP, which was a problem for a couple users as mentioned in the Slack channel. It is impossible to launch unless you have several side effects from installing other things on the system and potentially manually install an outdated tool and an outdated version of Chrome.

Of course, all of this is mitigated by running the command by hand and entering the result in the command line arguments. Which means the only way known to run this code is to avoid the code blocks that launch a browser. By extension, those code blocks that as far as I can inspect should not work on anyone's machine... well, maybe they work in some form of systems integration test or the context of everyone at CloudBees somehow.

In general, it is not desirable to launch and scrape in the current manner. In line with the thinking of #2651 I will check what APIs may exist to replace the function and chat in the Slack to make sure I am on the same page as you/your team.

Reply:
"no-longer-existing-or-supported" I am referring to ChromeDP here (link):